node/tests/SharedTests.ts

```diff
@@ -1400,7 +1413,7 @@ export function runBaseTests(config: {
         },
         config.timeout,
     );
-
+    // TO DO: fix this test.
```

Comment:
TODO reminder

Reply:
This has to updated when we change the return type of hscan. I would recommend reducing the number of lines at that time.

Reply:
I told Prateek to leave the test updates for when we work on `hscan` itself.